src/main/java/org/apache/parquet/hadoop/BdecParquetWriter.java

```diff
@@ -189,7 +190,7 @@ public boolean supportsBlockSize() {
 
     @Override
     public long defaultBlockSize() {
-      return (int) MAX_CHUNK_SIZE_IN_BYTES;
+      return maxChunkSizeInBytes;
```

Comment:
@sfc-gh-gdoci Any particular reason this was cast to int before?

Reply:
hmm, no, you can remove the cast.